.prettierrc

```
@@ -5,4 +5,4 @@
   "semi": true,
   "singleQuote": true,
   "trailingComma": "all"
-}
+}
```

let's revert this change.

```suggestion
}

```